app/views/chats/_chat_card.html.erb

```diff
@@ -1,6 +1,7 @@
 <div class="card bg-base-200 shadow-xl h-full place-content-between">
     <div class="card-body">
         <div class="card-title divider text-lg sm:text-sm underline"><%= link_to chat.url, chat %></div>
+        <div class="logs-link justify-centre text-sm underline"><%= "Logs", log_chat_path(chat), target: "_blank" %></div>
```

Comment:
You forgot to use the link_to helper.
In addition, I think this is more what you're attempting to accomplish. Change the entirety of line 4 to this
```erb
<%= link_to "Logs", log_chat_path(chat), target: "_blank", class: "justify-self-center underline text-center" %>
```

![image](https://github.com/hecksadecimal/newerparp/assets/20251491/b3267efb-5f95-4466-aa22-25bf60613ade)